specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -542,32 +542,16 @@
             "MS-AZR-0017P",
             "MS-AZR-0148P"
           ]
-        },
-        "additionalParameters": {
-          "description": "Additional, untyped parameters to support custom subscription creation scenarios.",
-          "type": "object",
-          "additionalProperties": {
-            "type": "object"
-          }
         }
       }
     },
     "ModernSubscriptionCreationParameters": {
       "description": "The parameters required to create a new subscription.",
-      "required": [
-        "displayName",
-        "billingProfileId",
-        "skuId"
```

Comment:
Why "skuId" is dropped from required as well?

Reply:
My bad. We want to remove only billingprofileId here